### PR TITLE
Canvas/cell templates

### PR DESCRIPTION
**Pull request by josiahhaswell**

### CODE REVIEW

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/canvas/Selector.java [hunk 1]
```diff
@@ -9,20 +9,17 @@
...
 
-
 @Getter
 @Setter
 @ToString
 @RootElement
 @EqualsAndHashCode
 @AllArgsConstructor
 @NoArgsConstructor
+@SuppressWarnings("PMD")
```
Comment (lisagumerman) [inline aire-ui/src/main/java/io/sunshower/zephyr/ui/canvas/Selector.java:19]: lol, what is there to be mad about in a 28 line class

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/canvas/StackCommandManager.java [hunk 2]
```diff
@@ -76,7 +76,6 @@ public void redo() {
   @Override
   public void apply(Action action) {
     undoStack.push(action);
-    action.apply(model);
```
Comment (lisagumerman) [inline aire-ui/src/main/java/io/sunshower/zephyr/ui/canvas/StackCommandManager.java:79]: Feels misnamed, given apply is no longer being called here

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/rmi/Parameters.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package io.sunshower.zephyr.ui.rmi;
+
+import lombok.val;
+
+public class Parameters {
+
+  @SuppressWarnings("unchecked")
+  public static String constructExpression(String methodName, Class<?>... values) {
+
+    val result = new StringBuilder("this.").append(methodName).append("(");
```
Comment (lisagumerman) [inline aire-ui/src/main/java/io/sunshower/zephyr/ui/rmi/Parameters.java:10]: I'm not sure if I hate this or I love it

File: aire-ui/src/main/resources/META-INF/resources/images/zephyr.svg [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="wind"
```
Comment (lisagumerman) [inline aire-ui/src/main/resources/META-INF/resources/images/zephyr.svg:1]: TODO: remove

File: aire-ui/src/test/java/io/sunshower/zephyr/ui/canvas/VertexTemplateTest.java [hunk 5]
```diff
@@ -16,27 +21,172 @@ void setUp() {
...
+            .hex("5F95FF")
+            .property("strokeWidth")
+            .number(1)
+            .property("fill")
+            .string("rgba(95,149,255,0.05)")
+            .tagName("rect")
+            .selector("body")
+            .attribute("image")
+            .property("xlink:href")
+            .string("https://gw.alipayobjects.com/zos/antfincdn/FLrTNDvlna/antv.png")
```
Comment (lisagumerman) [inline aire-ui/src/test/java/io/sunshower/zephyr/ui/canvas/VertexTemplateTest.java:58]: Lost opportunity to use a meme

**Review by lisagumerman — APPROVED** [review]

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/rmi/Parameters.java (hunk 3 above)
Comment (josiahhaswell) [reply aire-ui/src/main/java/io/sunshower/zephyr/ui/rmi/Parameters.java:10]: eh--works

**Pull request merged**